Testreglar/1.4.5/Nett/nett-1.4.5a.json

```diff
@@ -0,0 +1,131 @@
+{
+	"namn": "Nett-1.4.5a Bilde av tekst er ikkje brukt unødvendig 2023",
+	"id": "nett-1.4.5a",
+	"testlabId": 517,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nn",
+	"kravTilSamsvar": "<ul><li>Bilde av tekst kan visuelt tilpasses brukerens krav<ul><li>skrifttype, størrelse, farge og bakgrunn kan endres</li></ul></li><li>Bilde av tekst er brukt fordi en bestemt presentasjon av innholdet er avgjørende</li></ul>",
```

Comment:
```suggestion
	"kravTilSamsvar": "<p>Bilde av tekst er bare brukt dersom:</p><ul><li>det kan visuelt tilpasses brukerens krav<ul><li>skrifttype, størrelse, farge og bakgrunn kan endres</li></ul></li><li>en bestemt presentasjon av innholdet er avgjørende</li></ul>",
```

Reply:
Få tydeligere frem at det er bruken av bilde av tekst som er sentral for kravet, dagens krav til samsvar har bare info om unntakene.